Bottom: 90349400e7979291de16eadc38525f629093d67c
Top:    35a829f29483e3c8dfb3c5eb1487088e017a0139
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 14:37:02 +0800

autofs-5.0.7 - fix incorrect value reference in parse_line()

This chack should clearly be on the contents of key not the pointer value.


---

diff --git a/lib/defaults.c b/lib/defaults.c
index 1e89509..7c65387 100644
--- a/lib/defaults.c
+++ b/lib/defaults.c
@@ -167,7 +167,7 @@ static int parse_line(char *line, char **res, char **value)
 	while (*key && *key == ' ')
 		key++;
 
-	if (!key)
+	if (!*key)
 		return 0;
 
 	if (!(val = strchr(key, '=')))
